challenges/Job_Step_Viewer/README.md

```diff
@@ -0,0 +1,66 @@
+# Job Step Viewer and Summit Tour
+
+When running programs on a workstation, most of the time you can simply execute `./a.out` and wait for the results. On most HPC clusters, you must use a "batch scheduler" to request one or more compute nodes to run on and a "job launcher" to execute your program on the compute node(s) you were allocated. You submit jobs on Summit with IBM's [`jsrun` job launcher](../jsrun_Job_Launcher).
+
+The [Job Step Viewer](https://jobstepviewer.olcf.ornl.gov/) provides a graphical view of an application's runtime layout on Summit. It allows users to preview and quickly iterate with multiple jsrun options to understand and optimize job launch. It is a convenient tool that allows you to visualize and verify you are configuring your `jsrun` options in the way you expect.
+
+We recommend attempting this Challenge in conjunction with the [`jsrun` Job Launcher](../jsrun_Job_Launcher) challenge. You should go through the first example or two to get an understanding of how the `jsrun` command works, then re-run the examples, integrating the Job Step Viewer into your submission script. Then you can continue to use the Viewer as you go through the remaining examples and visualize your submissions as you go.
+
+## Job Step Viewer Usage
+
+First, request an allocation of nodes on Summit:
+
+```c
+$ bsub -P TRN001 -W 10 -nnodes 1 -J testing_job_step_viewer -U StudentsSC19 -Is $SHELL
+```
+
+Once your node is allocated, load the `job-step-viewer` module:
+
+```c
+$ module load job-step-viewer
+```
+
+Run a `jsrun` command by itself, or with an executable, such as the `Hello_jsrun` from the [`jsrun` Challenge](../jsrun_Job_Launcher). A URL will be printed out after a few seconds:
+
+```c
+$ export OMP_NUM_THREADS=2
+$ jsrun -n6 -c7 -g1 -a1
+
+
+https://jobstepviewer.olcf.ornl.gov/summit/315253-1
```

Comment:
Here, I might make a comment that this is an example you ran, but their link will be different.